students/morgan/session02/grid.py

```diff
@@ -0,0 +1,41 @@
+
+
+corner  = "+ "
```

Comment:
I like this!

Did you experiment with putting different characters in there to see how well it worked?

Reply:
a bit. it looks fine, just not as even. +, -, | just look more square on the grid.

Reply:
yup -- those probably are the best options -- but goodto knowthat the does work with other characters..